composer.json

```diff
@@ -2,6 +2,7 @@
     "name": "symfony/ux",
     "license": "MIT",
     "require-dev": {
-        "symfony/filesystem": "^5.2|^6.0"
+        "symfony/filesystem": "^5.2|^6.0",
+        "friendsofphp/php-cs-fixer": "^3.13"
```

Comment:
Will probably never be an issue here but might be better to use `php-cs-fixer/shim` to avoid any possible dep conflicts.

Reply:
Oh yeah, I forgot about this package.
Gonna change it ASAP.

Reply:
Done